package.json

```diff
@@ -92,6 +92,7 @@
     "dynamic-analysis": "npm run analyze-coverage && npm run check-coverage",
     "compile": "tsc -p src/ && webpack && node tools/fixupbundle.js",
     "test": "npm run compile && npm run all-tests && npm run static-analysis && npm run dynamic-analysis",
+    "prepublish": "npm run compile",
```

Comment:
See if I got this right @mikesherov.

* `dist/esprima.js` in the `files` field of package.json
* `dist` is **not** listed in `.npmignore`